EndlessLauncher/utility/Utils.cs

```diff
@@ -50,5 +51,20 @@ public static Process OpenUrl(string url, string args)
                 return null;
             }
         }
+
+        public static bool ActivateWindow(string cls, string win)
+        {
+            var window = FindWindow(cls, win);
+            if (window != IntPtr.Zero)
+            {
+                SetForegroundWindow(window);
+                if (IsIconic(window))
+                {
+                    OpenIcon(window);
+                }
```

Comment:
These functions are different to what was used in the Kolibri path before – but I guess they're equivalent?

Reply:
Not equivalent 100% but they serve the same purpose, it's tested yes.